index.html

```diff
@@ -317,33 +332,16 @@ <h2 class="title">About Myself</h2>
                       stroke-width="2"
                       stroke-linecap="round"
                       stroke-linejoin="round"
-                      class="feather feather-chevron-down"
+                      class="feather feather-chevron-up"
                     >
-                      <polyline points="6 9 12 15 18 9"></polyline>
+                      <polyline points="18 15 12 9 6 15"></polyline>
                     </svg>
-                  </label>
-                </summary>
-                <div class="summary-chevron-down">
-                  <svg
-                    xmlns="http://www.w3.org/2000/svg"
-                    width="24"
-                    height="24"
-                    viewBox="0 0 24 24"
-                    fill="none"
-                    stroke="currentColor"
-                    stroke-width="2"
-                    stroke-linecap="round"
-                    stroke-linejoin="round"
-                    class="feather feather-chevron-up"
-                  >
-                    <polyline points="18 15 12 9 6 15"></polyline>
-                  </svg>
-                </div>
-              </details>
-            </li>
+                  </div>
+                </details>
+              </li>
             <li>
               <details>
-                <summary>
+                <summary class="about-details-row">
                   Skills
 
```

Comment:
**[REQUIRED CHANGES]**

**NOTE: This change is not related to this part of your code. I just used it here to be able to add an inline comment. check below for the changes required in  the appropriate part of your code.**
**form**
- [Please, add the label attribute](https://github.com/danigalindo10/portfolio/blob/66d14ccdaf42a1cb81607f21da4a537e1d4f269e/index.html#L396-L402) to the form input and textarea to improve the accessibility and to ensure the has submitted the right input when the form is being processed in the server.
-  Please, check the video below and use this [WCAG Color contrast checker
chrome extension](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf?hl=en) to address the color contrast to improve the accessibility to visually impaired users:


https://user-images.githubusercontent.com/59397159/173433645-4275b366-13ea-4058-b962-1ef9fcb36963.mp4